sdk/helper/pluginutil/multiplexing_test.go

```diff
@@ -6,9 +6,83 @@ import (
 	"reflect"
 	"testing"
 
+	"google.golang.org/grpc"
+
 	"google.golang.org/grpc/metadata"
 )
 
+func TestMultiplexingSupported(t *testing.T) {
+	type args struct {
+		ctx  context.Context
+		cc   grpc.ClientConnInterface
+		name string
+	}
+
+	type testCase struct {
+		name    string
+		args    args
+		env     string
+		want    bool
+		wantErr bool
+	}
+
+	tests := []testCase{
+		{
+			name: "multiplexing is supported if plugin is not opted out",
+			args: args{
+				ctx:  context.Background(),
+				cc:   &MockClientConnInterfaceNoop{},
+				name: "plugin",
+			},
+			env:  "",
+			want: true,
+		},
+		{
+			name: "multiplexing is not supported if plugin is opted out",
+			args: args{
+				ctx:  context.Background(),
+				cc:   &MockClientConnInterfaceNoop{},
+				name: "optedOutPlugin",
+			},
+			env:  "optedOutPlugin",
+			want: false,
+		},
+		{
+			name: "multiplexing is not supported if plugin among one of the opted out",
+			args: args{
+				ctx:  context.Background(),
+				cc:   &MockClientConnInterfaceNoop{},
+				name: "optedOutPlugin",
+			},
+			env:  "firstPlugin,optedOutPlugin,otherPlugin",
+			want: false,
+		},
+		{
+			name: "multiplexing is supported if different plugin is opted out",
+			args: args{
+				ctx:  context.Background(),
+				cc:   &MockClientConnInterfaceNoop{},
+				name: "plugin",
+			},
+			env:  "optedOutPlugin",
+			want: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			t.Setenv(PluginMultiplexingOptOut, "optedOutPlugin")
```

Comment:
Wondering if this is supposed to set `tt.env`?

Reply:
Whoops you're right nice find